test/fixtures/jest/prefer-expect-assertions.fail.ts

```diff
@@ -0,0 +1,8 @@
+test("my test", () => {
+  expect.assertions("1");
+  expect(1).toEqual("foo");
+});
+
+test("my test", () => {
+  expect(1).toStrictEqual("foo");
+});
```

Comment:
This test uses double quote while the other one uses single quote.... please align this to showcase that the failure is not to do with quote choice.

Reply:
nice, thanks!

Reply:
fixed with https://github.com/ridedott/eslint-config/pull/367/commits/eac33b7cd680df7ab98581bb930710160827e779